provisioning/beetbox.sh

```diff
@@ -38,17 +38,20 @@ beetbox_setup() {
   echo "$BEET_USER ALL=(ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/$BEET_USER > /dev/null 2>&1
 
   # Install ansible.
-  sudo apt-get -qq -y install software-properties-common > /dev/null 2>&1
-  sudo apt-add-repository -y ppa:ansible/ansible > /dev/null 2>&1
-  sudo apt-get -qq update > /dev/null 2>&1
-  sudo apt-get -qq -y install ansible > /dev/null 2>&1
+  if [ ! -d "/etc/ansible" ]; then
+    sudo apt-get -qq -y install software-properties-common > /dev/null 2>&1
+    sudo apt-add-repository -y ppa:ansible/ansible > /dev/null 2>&1
+    sudo apt-get -qq update > /dev/null 2>&1
+    sudo apt-get -qq -y install ansible > /dev/null 2>&1
+  fi
 
   # Clone beetbox if BEET_HOME doesn't exist.
   if [ ! -d "$BEET_HOME" ]; then
```

Comment:
We actually don't need to wrap this in a condition anymore as the adhoc tasks are idempotent..